test/Store.test.js

```diff
@@ -6,33 +6,33 @@ import sinon from 'sinon';
 import { Store } from '../src';
 import { DISPATCH_TYPE, STATE_TYPE } from '../src/constants';
 
-describe('Store', function() {
-  beforeEach(function() {
+describe('Store', function () {
+  beforeEach(function () {
     // Mock chrome.runtime API
     global.chrome = {
       runtime: {
-        connect: function(opts) {
+        connect( /* opts */ ) {
```

Comment:
My vote is to remove the args here since we say `// Mock chrome.runtime API` and it's very easy to look that up. I think the redux case is easier to mess up since there are so many functions returning functions.